### PR TITLE
Updated readme, Added projects

### PR DESCRIPTION
**Pull request by MaribelleHGomez**
Description about what this pull request does.

Updated readme and added new GDC projects

Please make sure to follow the [DEV guidelines](https://gen3.org/resources/developer/dev-introduction/) before asking for review.

### Improvements
- Added new GDC projects to open bucket exception list
- Updated readme to reflect more recent workflow process and clarity

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,56 +23,145 @@ aws_secret_access_key=xxxxxxxx
...
+Both the `sync manifest` and `redaction manifest` have the same format as described below:
 ```
-gen3 runjob jobs/aws-bucket-replicate-job.yaml GDC_BUCKET_NAME gdcbackup MANIFEST_S3 s3://tests/manifest THREAD_NUM 100 LOG_BUCKET log_bucket CHUNK_SIZE 50
+id	file_name	md5	size	state	project_id	baseid	version	release	acl	type	deletereason	url
+ada53c3d-16ff-4e2a-8646-7cf78baf7aff	ada53c3d-16ff-4e2a-8646-7cf78baf7aff.vcf.gz	ff53a02d67fd28fcf5b8cd609cf51b06	137476	released	TCGA-LGG	6230cd6d-8610-4db4-94f4-2f87592c949b	1	12.0	[u'phs000xxx']	active		s3://xxxx/ada53c3d-16ff-4e2a-8646-7cf78baf7aff
+2127dca2-e6b7-4e8a-859d-0d7093bd91e6	2127dca2-e6b7-4e8a-859d-0d7093bd91e6.vep.vcf.gz	c4deccb43f5682cffe8f56c97d602d08	136553	released	TCGA-LGG	6a84714f-2523-4424-b9cb-c21e8177cd0f	1	12.0	[u'phs000xxx']	active		s3://xxxx/2127dca2-e6b7-4e8a-859d-0d7093bd91e6/2127dca2-e6b7-4e8a-859d-0d7093bd91e6.vep.vcf.gz
 ```
 
-### GOOGLE sync
-- Put the `gcloud-creds-secret` and `dcf_dataservice_settings` into the `dcf_dataservice` folder.
```
Comment (BinamB) [inline README.md:56]: Why did you remove this. Its not part of the current process but this instruction is required for initial set up

File: README.md [hunk 2]
```diff
@@ -45,56 +23,145 @@ aws_secret_access_key=xxxxxxxx
...
+
 ```
-GDC_TOKEN = "TOKEN"
-INDEXD = {
+gcs_object_size	gcs_object_time_stamp	md5sum	acl	gcs_object_url	gdc_uuid
+19490957	2016-03-28T21:51:54Z	bd7b6da28d89c922ea26d145aef5387e	phs000xxx	gs://5aaxxxxxxxx/tcga/PAAD/DNA/AMPLICON/BI/ILLUMINA/C1748.TCGA-LB-A9Q5-01A-11D-A397-08.1.bam	e5c9f17d-8f66-4089-ac70-2a5cde55450f
+```
+
+### Copy Manifests
+- Download released manifest files from box https://uchicago.app.box.com/folder/9184104361?s=8l1tagrjbb183qxys0o6rji3rlskzu85 (Downloading and saving the files under named folder DCF makes copying files to DCF prod vm easier)
```
Comment (BinamB) [inline README.md:44]: We probably dont want to put the box link in a public facing document

File: README.md [hunk 3]
```diff
@@ -45,56 +23,145 @@ aws_secret_access_key=xxxxxxxx
...
+{
+"GDC_TOKEN": "TOKEN",
+"INDEXD": {
     "host": "https://nci-crdc.datacommons.io/index/index",
     "version": "v0",
     "auth": {"username": "gdcapi", "password": "xxxxx"}
 }
+}
 ```
-- Also put the `GDC_project_map.csv` into `dcf_dataservice` folder.
```
Comment (BinamB) [inline README.md:66]: same as above. Need these instructions for first time setup

File: README.md [hunk 4]
```diff
@@ -45,56 +23,145 @@ aws_secret_access_key=xxxxxxxx
...
+### If pass
+Replace True with False in previous cmd
 
+## Redaction
+Upload redaction manifest to s3 and run job
+
+### Dry Run
+`gen3 runjob remove-objects-from-clouds-job DRY_RUN True RELEASE DR<release number> MANIFEST_S3 s3://data-refresh-manifest/<obsolete manifest file name> LOG_BUCKET data-refresh-output IGNORED_FILE_S3 s3://data-refresh-manifest/ignored_files_manifest.csv`
+
+### If pass
```
Comment (BinamB) [inline README.md:160]: dry run to check if anything needs to be redacted. If dry run finds nothing, no need to run the real thing. The job fails if yuou run when you have nothing to redact

File: README.md (hunk 1 above)
Comment (MaribelleHGomez) [reply README.md:56]: if you read you'll find it

File: README.md (hunk 2 above)
Comment (MaribelleHGomez) [reply README.md:44]: oh you're right!
